rpm-4.8.0-htonll-fix.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/lib/header.c b/lib/header.c
```

Comment:
Ok, perfect. I've added your patch to https://github.com/cms-externals/rpm/tree/cms/v4.8.0. Will soon adapt this PR to pick up sources from there.